tests/unit/test_index.py

```diff
@@ -122,11 +122,11 @@ def warning(self, *args, **kwargs):
 @pytest.mark.parametrize(
     ("location", "trusted", "expected"),
     [
-        ("http://pypi.python.org/something", [], True),
-        ("https://pypi.python.org/something", [], False),
-        ("git+http://pypi.python.org/something", [], True),
-        ("git+https://pypi.python.org/something", [], False),
-        ("git+ssh://git@pypi.python.org/something", [], False),
+        ("http://pypi.org/something", [], True),
+        ("https://pypi.org/something", [], False),
+        ("git+http://pypi.org/something", [], True),
+        ("git+https://pypi.org/something", [], False),
+        ("git+ssh://git@pypi.org/something", [], False),
```

Comment:
These are dummy URLs, in the sense that we don't fetch from them (after all, we also use example.com). While I understand you probably just got them by search and replace, is this change necessary?

Reply:
Nah, I can keep the old URLs. I just changed them for complete-ness sake.

Reply:
Actually, if this isn't landing in 10.0.0 I'll likely just keep these, just because I think in N years we'll be scratching our heads about this in the same way we were with ``b.pypi.python.org`` :)